packages/@coorpacademy-components/src/atom/button-link/index.tsx

```diff
@@ -9,15 +9,24 @@ import style from './style.css';
 
 const getButtonContent = (icon?: IconType, label?: string) => {
   const {type, faIcon, position} = icon || {type: '', position: ''};
+  const Icon = type && ICONS[type];
+
+  if (!Icon && !faIcon) {
+    return (
+      <div className={style.buttonContent}>
+        <span className={style.label}>{label}</span>
+      </div>
+    );
+  }
+
   const {name, color, backgroundColor, size} = faIcon || {
     name: '',
     color: '',
     backGroundColor: '',
     size: 0
   };
-  const Icon = type && ICONS[type];
 
-  const faIconComponent = (
+  const iconComponent = faIcon ? (
     <FaIcon
       {...{
         iconName: name,
```

Comment:
using `faIcon.name` and so on, you wont need destructuring faIcon and fallbacl on line before

Reply:
(in other words: remove `const {name, color, backgroundColor, size} = faIcon || {` and `iconName: faIcon.name, color: faIcon.color, ...`
(fallback might not be necessary)